employee-rostering-backend/src/test/java/org/optaweb/employeerostering/optawebemployeerosteringbackend/OptawebEmployeeRosteringBackendApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package org.optaweb.employeerostering.optawebemployeerosteringbackend;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class OptawebEmployeeRosteringBackendApplicationTests {
+
+	@Test
+	public void contextLoads() {
```

Comment:
What does this test do? It is empty, so does it just test that it compiles correctly? Also, the file name should end in `Test` not `Tests`. If this test isn't testing anything, then please remove it.

Reply:
Sorry, this was autogenerated in the Spring initializer; I'll remove it until I add meaningful tests.